### PR TITLE
[BUG#1] [Header]  header fix

### PR DESCRIPTION
**Pull request by pashamed**
https://trello.com/c/ALUeTqFT/62-bug1-header-the-header-does-not-match-the-design
image

### CODE REVIEW

**Comment by pashamed** [comment]
[[BUG#1] [Header] The Header does not match the design](https://trello.com/c/ALUeTqFT/62-bug1-header-the-header-does-not-match-the-design)

File: frontend/src/app/modules/main/main-page/main-page.component.html [hunk 1]
```diff
@@ -1,7 +1,6 @@
+<app-header></app-header>
```
Comment (tatianahutii) [inline frontend/src/app/modules/main/main-page/main-page.component.html:1]: header should be not only in the main page

**Review by tatianahutii — CHANGES_REQUESTED** [review]

File: frontend/src/app/modules/login/log-in-page/log-in-page.component.html [hunk 2]
```diff
@@ -1,3 +1,4 @@
+<app-header></app-header>
```
Comment (tatianahutii) [inline frontend/src/app/modules/login/log-in-page/log-in-page.component.html:1]: header should be added once, for all pages

**Review by tatianahutii — CHANGES_REQUESTED** [review]

**Review by pashamed — COMMENTED** [review]
adde header globaly

File: frontend/src/app/app.component.ts [hunk 3]
```diff
@@ -4,7 +4,7 @@ import { SpinnerService } from '@core/services/spinner.service';
 
 @Component({
     selector: 'app-root',
-    template: '<app-loading-spinner></app-loading-spinner><router-outlet></router-outlet>',
+    template: '<app-header></app-header><app-loading-spinner></app-loading-spinner><router-outlet></router-outlet>',
```
Comment (tatianahutii) [inline frontend/src/app/app.component.ts:7]: not here=)

**Review by tatianahutii — CHANGES_REQUESTED** [review]

File: frontend/src/app/app.component.ts (hunk 3 above)
Comment (pashamed) [reply frontend/src/app/app.component.ts]: hope this is the right place =)

**Review by apofis96 — CHANGES_REQUESTED** [review]
Hint: header should be be added only once before router outlet

**Review by tatianahutii — APPROVED** [review]

**Pull request merged**